src/OpenDebugAD7/OpenDebugAD7.csproj

```diff
@@ -98,34 +61,25 @@
     </ProjectReference>
   </ItemGroup>
   
-  <ItemGroup Label="OpenDebugAD7 Drop Files">
-    <DropSignedFile Include="$(OutputPath)\OpenDebugAD7.exe" />
-    <DropUnsignedFile Include="$(OutputPath)\OpenDebugAD7.exe.config" />
-    <DropThirdPartySignedFile Include="$(OutputPath)\Newtonsoft.Json.dll" />
-    <DropUnsignedFile Include="$(OutputPath)\Microsoft.VisualStudio.Shared.VSCodeDebugProtocol.dll" />
-    <DropUnsignedFile Include="$(OutputPath)\Microsoft.VisualStudio.Interop.dll" />
-  </ItemGroup>
-  <ItemGroup Label="OpenDebugAD7 Drop Files for XPlat" Condition="'$(IsXPlat)' == 'false'">
-    <!-- 
-      Any libraries that exist in VS but are needed for VS OpenDebugAD7 
-      scenarios will need to be added via VS Setup files.
-    -->
-    <DropUnsignedFile Include="$(OutputPath)\cppdbg.ad7Engine.json" />
-  </ItemGroup>
-  <Target Name="GetDropFiles" BeforeTargets="DropFiles">
-    <ItemGroup>
-      <DropUnsignedFile Include="$(OutputPath)\OpenDebugAD7.pdb" />
-    </ItemGroup>
-  </Target>
-  <ItemGroup>
-    <None Include="App.VSCode.config" />
-    <None Include="App.VS.config" />
-  </ItemGroup>
   <ItemGroup>
     <None Include="cppdbg.ad7Engine.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+
+  <ItemGroup Label="Drop Files">
+    <DropSignedFile Include="$(MIDefaultOutputPath)\Microsoft.MICore.dll" />
+    <DropUnsignedFile Include="$(MIDefaultOutputPath)\Microsoft.MICore.pdb" />
+    <DropSignedFile Include="$(MIDefaultOutputPath)\Microsoft.MIDebugEngine.dll" />
+    <DropUnsignedFile Include="$(MIDefaultOutputPath)\Microsoft.MIDebugEngine.pdb" />
+    <DropSignedFile Condition="$([MSBuild]::IsOSPlatform('windows'))" Include="$(OutputPath)\OpenDebugAD7.exe" />
```

Comment:
>Include="$(OutputPath)\OpenDebugAD7.exe" /> [](http://example.com/codeflow?start=70&length=43)

I believe you want to leave the exe unsigned (it is already signed) but you need to drop the .dll now.